src/ts/components/settingspanel.ts

```diff
@@ -20,15 +21,29 @@ export interface SettingsPanelConfig extends ContainerConfig {
    * Default: 3 seconds (3000)
    */
   hideDelay?: number;
+
+  /**
+   * Flag to specify if there should be an animation when switching SettingsPanelPages.
+   * Default: true
+   */
+  pageTransitionAnimation?: boolean;
 }
 
 /**
  * A panel containing a list of {@link SettingsPanelItem items} that represent labelled settings.
  */
 export class SettingsPanel extends Container<SettingsPanelConfig> {
 
-  private static readonly CLASS_LAST = 'last';
+  private static readonly ACTIVE_PAGE_CLASS = 'active';
```

Comment:
Statically defined class string are always prefixed with `CLASS_` thoughout this project, not suffixed.